### DISCUSSION TITLE
Public archive 프로젝트 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#12] [Q&A] [answered]
저번 주에 온라인 게임잼에 참여해서 프로젝트를 만들고 2박 3일동안 해커톤처럼 게임을 개발했습니다..!  

게임잼이 끝나고 프로젝트도 이후 개발일정이 없어서 private에서 Public archive로 전환했습니다.  

제가 이해한게 맞다면 onlyread형식으 레포가 맞을 까요?

[Garbage_Game](https://github.com/fkdl0048/Garbage_Game)
Public archive

**Comment by jongfeel** [comment, marked as answer]
제가 들어갔을 때 이런 메시지가 나오는 걸 봐서 맞는 것 같습니다

![image](https://user-images.githubusercontent.com/17442457/197669326-a2d3cdce-9467-420a-a845-1d115c0fc6dd.png)

**Reply by fkdl0048** [reply]
감사합니다.. 앞으로 종료된 프로젝트는 Public archive로 전환해서 관리하겠습니다!
